src/main/java/no/sikt/sws/RequestUtil.java

```diff
@@ -28,4 +30,16 @@ public static String getWorkspace(RequestInfo request) {
             return fullScope.replaceFirst(SCOPE_IDENTIFIER + "/", "");
         }).orElseThrow();
     }
+
+    public static String addWorkspace(String body, String workspace, String index) {
+        if (body == null) {
+            return null;
+        }
+
+        return attempt(() -> {
+            var strippedIndex = Arrays.stream(index.split("/")).findFirst();
+            return body.replaceAll(index, workspace + "-" + strippedIndex);
```

Comment:
I can see alot of issues with this if the index is named somethigng short..  you should see if there is a better way of doing it. Like figuring out the excact usecases (json-bodies)